src/participant/constants.ts

```diff
@@ -0,0 +1 @@
+export const CHAT_PARTICIPANT_ID = 'mongodb.participant';
```

Comment:
Curious, why would you want to store it separately? How is it different from e.g. CHAT_PARTICIPANT_MODEL or other constants that we have around?

Reply:
Only moved it separate to avoid a circular dependency, it's used in two places. I imagine more things will move here over time.